sorc/link_workflow.sh

```diff
@@ -75,7 +75,8 @@ case "${machine}" in
   "hercules") FIX_DIR="/work/noaa/global/kfriedma/glopara/fix" ;;
   "jet")      FIX_DIR="/lfs4/HFIP/hfv3gfs/glopara/git/fv3gfs/fix" ;;
   "s4")       FIX_DIR="/data/prod/glopara/fix" ;;
-  "gaea")     FIX_DIR="/gpfs/f5/ufs-ard/world-shared/global/glopara/data/fix" ;;
+  "gaea")     FIX_DIR="/gpfs/f5/epic/proj-shared/global/glopara/data/fix" ;;
```

Comment:
@weihuang-jedi Did the Gaea team approve of the change in the fix_dir location for GAEA?

Reply:
@aerorahul I did not change that line myself. Those lines are coming up when I sync my branch with EMC repo (or because it did not sync well with EMC repo). 
I'll try to sync mine with EMC repo more often to see if such thing can be avoid.